provider/storage_benchmark.py

```diff
@@ -392,6 +392,7 @@ def __init__(self, params, vm):
         self.cfg = self.cfg_map[self.os_type](params, self.session)
         for method in self.cfg.setup_orders:
             method(self)(*self.cfg.setups[method])
+        self.run("-v")
```

Comment:
Could you possibly remove this line, because 
1. we cannot run fio when we create a Fio object with generate_instance, as in the test cases, we always create a Fio object, and then explicitly call fio.run() to run it, so if we run fio here, we run it again in those test cases.
2. Not all tests need the verbose output of fio

Reply:
updated